docs/python/tutorials/integrating-dbos.md

```diff
@@ -0,0 +1,190 @@
+---
+sidebar_position: 10
+title: Adding DBOS To Your App
+pagination_next: null
+---
+import InstallNode from '/docs/partials/_install_node.mdx';
+import LocalPostgres from '/docs/partials/_local_postgres.mdx';
+
+### Using DBOS Transact
+
+This guide shows you how to add the open source [DBOS Transact](https://github.com/dbos-inc/dbos-transact-py) library to your application to **durably execute** it and make it resilient to any failure.
```

Comment:
Also, maybe move this above the "Using DBOS Transact" title and say that you can optionally deploy it to DBOS Cloud.